packages/oui-radio/src/radio.controller.js

```diff
@@ -13,11 +14,11 @@ export default class {
     $postLink () {
         // Sometimes the digest cycle is done before dom manipulation,
         // So we use $timeout to force the $apply
-        this.$timeout(() =>
+        this.$timeout(() => {
```

Comment:
This is equivalent of what was already there.

Reply:
removed